xml/System.Net.Http/SocketsHttpHandler.xml

```diff
@@ -699,7 +699,7 @@ For example, if the value is 64, then 65,536 bytes are allowed for the maximum r
         <ReturnType>System.Net.IWebProxy</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets or sets the custom proxy when the <see cref="P:System.Net.Http.SocketsHttpHandler.UseProxy" /> property is <see langword="true" />.</summary>
+        <summary>Gets or sets the custom proxy when the?<see cref="P:System.Net.Http.SocketsHttpHandler.UseProxy" />?property is <see langword="true" />.</summary>
```

Comment:
What does the question marks mean?

Reply:
This change should be reverted.

Reply:
Thanks for noticing. I noticed another one which I deleted but didn't catch this one. I have to check if the tool has a bug.

Reply:
```suggestion
        <summary>Gets or sets the custom proxy when the?<see cref="P:System.Net.Http.SocketsHttpHandler.UseProxy" /> property is <see langword="true" />.</summary>
```